src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -239,7 +240,8 @@
 						"as this event is intended to support changing the hatched mob and its chance to hatch, but Skript does not yet support that.")
 				.examples("")
 				.since("1.0");
-		// TODO improve - on fish [of %entitydata%] (and/or itemtype), on reel, etc.
+		//TODO improve - on fish [of %entitydata%] (and/or itemtype), on reel, etc.
+		//Note: Should be something like RandomSK "[on] fishing state of %fishingstate%"
```

Comment:
I wouldn't say this a good design, adding specific events for each state could be an option too.

Reply:
Pull requests should not add comments describing future plans on Skript development. A proper way would be opening an issue.